python-backend/app/api/mines/custom_reqparser.py

```diff
@@ -0,0 +1,18 @@
+from flask_restplus import reqparse
+from werkzeug.exceptions import BadRequest
+
+DEFAULT_MISSING_REQUIRED = 'Missing required argument.'
+
+class CustomReqparser():
+
+    def __init__(self):
+        parser_instance = reqparse.RequestParser(trim=True)
+        self.parser = parser_instance
+        self.add_argument = parser_instance.add_argument
+
+    def parse_args(self):
+        try:
+            data = self.parser.parse_args()
+        except BadRequest:
+            raise BadRequest(DEFAULT_MISSING_REQUIRED)
+        return data
```

Comment:
Should there be one CustomReqparser that lives in utils folder? eventually we will need to package that utils folder for reuse anyways.

Reply:
Yes, I think so. But I don't understand what rules we have for our microservices. My only previous experience with microservices used Go and kept each service in a separate repo (which Go could easily require in), so I'm not clear on where I can/can't put shared code.

I figured I had to put an instance in each namespace at this point.